node-webservice/react-client/src/Components/GridCell.jsx

```diff
@@ -1,29 +1,51 @@
 import React, { Component } from 'react';
+import Block from "../Images/block.PNG";
+import Center from "../Images/center.PNG";
+import SpikeNorth from "../Images/spikeNorth.PNG";
```

Comment:
I worry a bit about how this will work when scaling up the number of available parts. Not necessary to change right now, but consider rotating the image using css https://www.w3schools.com/cssref/css3_pr_transform.asp